x-pack/legacy/plugins/uptime/server/lib/adapters/monitors/extract_filter_aggs_results.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { OverviewFilters } from '../../../../common/runtime_types';
+
+type SupportedFields = 'locations' | 'ports' | 'schemes' | 'tags';
+
+export const extractFilterAggsResults = (
```

Comment:
If we're creating a separate file for a function that is called from exactly one place the code gets harder to follow IMHO. In general, from a style standpoint IMHO it makes more sense for a function like this to exist in the same file from which its called.

Reply:
I don't really have any issue with it living in the same file; it felt cleaner to me to have it in a dedicated location but I'm 👍 on moving it back. https://github.com/elastic/kibana/pull/50837/commits/967adf54ecf261a28bcb46ec55329af966ca7621